README.md

```diff
@@ -9,15 +9,15 @@
 
 ## Main features
 
-1. **[Programming exercises](https://docs.artemis.ase.in.tum.de/user/exercises/programming/)** with version control, automatic individual feedback (and assessment) based on test cases and static code analysis (executed using continuous integration). 
+1. **[Programming exercises](https://docs.artemis.ase.in.tum.de/user/exercises/programming/)** with version control, automatic individual feedback (and assessment) based on test cases and static code analysis (executed using continuous integration).
    * **Instant**: Students receive immediate and individual feedback on submissions. Instructors can customize feedback messages easily, hide feedback during the working time (e.g., with hidden tests)
    * **Interactive:** Instructors integrate interactive instructions based on tasks and UML diagrams directly into the dynamic problem statements. They can define hints for difficult exercise parts.
    * **Independent**: Instructors can customize programming exercises to support any programming language. To simplify the setup, Artemis includes sophisticated templates for the most common languages (e.g., Java, Python, C, Haskell, Kotlin, VHDL, Assembler, Swift, Ocaml, ...)
    * **Local and online**: Students can participate locally in their preferred IDE or in the online code editor (without any setup)
    * **Manual reviews** are possible directly in the online code editor or in [Orion](https://github.com/ls1intum/Orion)
-   * **Policies**: Instructors can define submission policies (e.g., penalties after 10 attempts) to prevent that students try out all possibilities without thinking.
+   * **Policies**: Instructors can define submission policies (e.g., penalties after 10 attempts) to prevent students from trying out all possibilities without thinking.
    * **Grading**: Instructors have many options to configure grading, analyze the results based on tests and static code analysis categories and re-evaluate the results
-   * **Secure**: Test cases and student code run in Docker environments on build agents. Test frameworks such as [Ares](https://github.com/ls1intum/Ares) simplify the creation of structural and dynamic tests and prevent that students can cheat. 
+   * **Secure**: Test cases and student code run in Docker environments on build agents. Test frameworks such as [Ares](https://github.com/ls1intum/Ares) simplify the creation of structural and dynamic tests and prevent students from cheating.
```

Comment:
For me the old formulation sounds better here (Ares does not prevent cheating attempts, it makes the attempts non sucessfull).

Reply:
would it be better then "that catch possible student attempts of cheating"?

Reply:
No, catch would mean getting notifications or other information about it. Here it is just a security framework that denies attempts.

Reply:
I disagree, the new formulation sounds better to me.